src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpSender.php

```diff
@@ -33,7 +33,7 @@ public function __construct(EncoderInterface $messageEncoder, Connection $connec
     /**
      * {@inheritdoc}
      */
-    public function send($message)
+    public function send($message): void
```

Comment:
I’m not fond of the idea of enforcing `void` as a result. Senders might return something later on, we shouldn’t close that door IMHO.

Reply:
Removing void should be ok for the bc policy. See https://github.com/symfony/symfony-docs/pull/9717

Reply:
Fair enough.